libethcore/Precompiled.cpp

```diff
@@ -99,48 +99,77 @@ ETH_REGISTER_PRECOMPILED(identity)(bytesConstRef _in)
 
 // Parse _count bytes of _in starting with _begin offset as big endian int.
 // If there's not enough bytes in _in, consider it infinitely right-padded with zeroes.
-bigint parseBigEndianRightPadded(bytesConstRef _in, size_t _begin, size_t _count)
+bigint parseBigEndianRightPadded(bytesConstRef _in, bigint _begin, bigint _count)
 {
-	if (_begin > _in.count())
+	if (_begin > numeric_limits<size_t>::max())
```

Comment:
Even if `_begin > numeric_limits<size_t>::max()`, when `_begin` is one more than the max value, `begin` would be zero.  So I leave these tests.